src/platform/common/platform/fileSystem.node.ts

```diff
@@ -80,6 +70,9 @@ export class FileSystem extends FileSystemBase implements IFileSystemNode {
     public async localDirectoryExists(dirname: string): Promise<boolean> {
         return this.localPathExists(dirname, FileType.Directory);
     }
+    public override async deleteLocalFile(path: string): Promise<void> {
+        await fs.unlink(path);
```

Comment:
The base method relies on VS Code API, without this change the files never get deleted when VS Code is shutting down on reloading, meaning the VSC API is slow and as mentioned earlier this is unnecessary as we're dealing with local files, again using native node API is more efficient (proven by the fact that without this change things don't work as expected - when reloading vscode or shutting it down)